<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectCard</class>
 <widget class="QWidget" name="ProjectCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>76</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>412</width>
    <height>76</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>412</width>
    <height>76</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#container {
    border: 1px solid #D9D9D9;
    border-radius:  8px;
}

#container:hover {
	background-color: #D6ECFF;
}

#projectInfo {
    font-family: Open Sans;
    font-weight: 500;
    line-height: 24px;
    letter-spacing: 0%;
    font-size: 16px;
    color: #666666;
}

#widget {
    background-color: transparent;
}

#projectNameLabel  {
    font-family: Open Sans;
    font-weight: 500;
    line-height: 26px;
    letter-spacing: 0%;
    font-size: 18px;
    color: black;
}

#menuButton {
    background: transparent;
    border: none;
}

#projectIcon {
    background: transparent;
    border: none;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="container" native="true">
     <property name="minimumSize">
      <size>
       <width>410</width>
       <height>70</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QToolButton" name="projectIcon">
        <property name="icon">
         <iconset resource="../../../../Resources/Resources.qrc">
          <normaloff>:/images/header-project.svg</normaloff>:/images/header-project.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="projectNameLabel">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Lorem Ipsum ath</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="menuButton">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../../Resources/Resources.qrc">
                <normaloff>:/images/ellipsis-vertical.svg</normaloff>:/images/ellipsis-vertical.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="projectInfo"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
